docs/docfx/articles/toc.yml

```
@@ -40,3 +40,5 @@
   href: packages-refs.md
 - name: Diagnosing proxy issues
   href: diagnosing-yarp-issues.md
+- name: AB Testing
```

Does it support `/ ` in the name?
```suggestion
- name: A/B Testing
```